Makefile

```diff
@@ -185,8 +185,7 @@ setup-hooks:
 	pip install -c constraints.txt pre-commit
 	pre-commit install
 
-typecheck:
-	./check_types.sh
+typecheck: lint
```

Comment:
could you restore the previous behavior (e.g. just type-checking but not linting). It's nice to be able to check types on unformatted code.

Reply:
Done.